examples/fmri_ants_openfmri.py

```diff
@@ -150,11 +160,12 @@ def create_reg_workflow(name='registration'):
     register.connect(stripper, 'out_file', convert2itk, 'reference_file')
     """
     Compute registration between the subject's structural and MNI template
-    This is currently set to perform a very quick registration. However, the
-    registration can be made significantly more accurate for cortical
-    structures by increasing the number of iterations
-    All parameters are set using the example from:
-    #https://github.com/stnava/ANTs/blob/master/Scripts/newAntsExample.sh
+
+        * All parameters are set using the example from: \
+          `newAntsExample.sh <https://github.com/stnava/ANTs/blob/master/Scripts/newAntsExample.sh>`_
+        * This is currently set to perform a very quick registration. However,\
+          the registration can be made significantly more accurate for cortical\
+          structures by increasing the number of iterations.
```

Comment:
This is part of a docstring. It doesn't look very good. https://387-59325098-gh.circle-artifacts.com/0/home/circleci/work/docs/users/examples/fmri_ants_openfmri.html

I'd say revert this block.